Assets/_Scripts/StageInfo.cs

```diff
@@ -29,45 +24,78 @@ public class StageInfo : MonoBehaviour
     public Player GetHeroPlayer() {
         return _heroPlayer;
```

Comment:
Do we really need reference to hero player here? We already have reference in party.

Reply:
It's just for shorthand